src/components/LoginForm.tsx

```diff
@@ -0,0 +1,37 @@
+'use client'
+import { addAuthUser } from '@/store/actions/auth-action';
+import { RootState } from '@/store/store';
+import * as React from 'react'
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { Dispatch } from 'redux';
+
+const LoginForm: React.FC = () => {
+  const [userId, setUserId] = React.useState('')
+  
+  const authState = useSelector((state: RootState) => state.reducerAuth.auth)
+
+  const dispatch: Dispatch<any> = useDispatch()
+
+  const login = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault()
+    console.log('data before login', authState);
```

Comment:
Always Remove the `console.log`, in the code that you merge to main

Reply:
Done, And I will by aware for the next time.